core/pyos.py

```diff
@@ -282,6 +282,14 @@ def InputAvailable(fd):
     return len(r) != 0
 
 
+def WaitForInputs(fd):
```

Comment:
fd implie a single descriptor -- I would call this `fd_list`

Some other names could be `WaitToRead(fd_list)` or `WaitForReading(fd_list)`